tests/elements-regression/tests/elements-regression.test.js

```diff
@@ -39,25 +55,26 @@ test.describe( 'Elementor regression tests with templates for CORE', () => {
 			await wpAdminPage.openNewPage();
 			await editorPage.closeNavigatorIfOpen();
 			await editorPage.loadTemplate( filePath );
-			if ( 'video' === widgetType ) {
-				await editorPage.waitForVideoLoaded();
-			}
+			await editorPage.waitForIframeToLoaded( widgetType );
 
 			const widgetCount = await editorPage.getWidgetCount();
 			const widgetIds = [];
 			for ( let i = 0; i < widgetCount; i++ ) {
 				const widget = editorPage.getWidget().nth( i );
 				const id = await widget.getAttribute( 'data-id' );
+				await expect( widget ).not.toHaveClass( /elementor-widget-empty/ );
 				widgetIds.push( id );
 				await editorPage.waitForElementRender( id );
-				await expect( widget ).toHaveScreenshot( `${ widgetType }_${ i }.png`, { maxDiffPixels: 100 } );
+				await expect( widget ).toHaveScreenshot( `${ widgetType }_${ i }.png`, { maxDiffPixels: 100 }, { timeout: 10000 } );
 			}
+
+			const response = page.waitForResponse( /http:\/\/(.*)\/wp-content\/uploads(.*)/g );
 			await editorPage.publishAndViewPage();
 			await editorPage.waitForElementRender( widgetIds[ 0 ] );
-			if ( 'video' === widgetType ) {
-				await editorPage.waitForVideoLoaded( true );
-			}
-			await expect( page.locator( EditorSelectors.container ) ).toHaveScreenshot( `${ widgetType }_published.png`, { maxDiffPixels: 100 } );
+			await editorPage.waitForIframeToLoaded( widgetType, true );
+			await response;
+
+			await expect( page.locator( EditorSelectors.container ) ).toHaveScreenshot( `${ widgetType }_published.png`, { maxDiffPixels: 100 }, { timeout: 10000 } );
```

Comment:
@dmaziar Can you please check this?

<img width="664" alt="Screenshot 2023-04-25 at 15 12 13" src="https://user-images.githubusercontent.com/94352322/234287205-e3fc05f8-0163-4963-8ffe-10c3acf246d0.png">

Reply:
This shows that the Google maps test was flaky-can you please check it out?

Reply:
fixed